.github/actions/lint/lint_links.py

```diff
@@ -30,12 +39,63 @@ def main():
 
 
 ## HELPER FUNCTIONS
-def get_markdown_files(root_dir):
+def parseInputArguments():
+    """
+    Parsing arguments for the root directory and config file.
+
+    Returns:
+        ArgumentParser: An argument parser object used for accessing command line arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-r', '--root', default=ROOT_DEFAULT, nargs='?', type=str, help='Path to root directory where linting begins')
+    parser.add_argument('-c', '--config', default=CONFIG_DEFAULT, nargs='?', type=str, help='Path to a JSON configuration file specifying ignore files and patterns')
+    args = parser.parse_args()
+    return args
+
+
+def get_ignore_patterns(config_file):
+    """
+    Obtain a list of patterns to ignore specified in the config file whose path is specified in lint.yml.
+
+    Args:
+        config_file (str): The path of the config file relative to the root.
+
+    Returns:
+        List[str]: A list of regex patterns to ignore when performing linting.
+    """
+    ignore_patterns = []
+    if os.path.isfile(config_file):
```

Comment:
If no config file is found, I suggest printing a warning to `stderr` that no file was found in the else branch. If someone put a path to a config file but made a mistake, it would be very confusing as to why it didn't work without any feedback.

```python
import sys

# Print to stderr
print("Hello World!", file=sys.stderr)
```

Reply:
Same with Line 85 below.